build.gradle

```diff
@@ -11,16 +11,13 @@ plugins {
     id "java-library"
     id "checkstyle"
     id "signing"
-    id "com.github.johnrengelman.shadow" version "4.0.4"
```

Comment:
We're not actually using Shadow for this; we need to expose the Consul client library as a transitive dependency, because its types are exposed in our API (and also so the user can override the dependency version if they want to). The Shadow stuff here was just unused copypasta.